### DISCUSSION TITLE
Storybook different company icons for dark and light mode

### DISCUSSION THREAD

**Discussion by pedaars** [storybookjs/storybook#29218] [Help]
### Summary

We have our storybook ui as default just using the users system preferences as to whether it shows in light mode or dark mode. I have added our company logo top left to replace the default storybook logo but I cant see how I can set a different logo for dark and light theme. Can anyone point me to any documentation on this please?

**Comment by Sidnioulz** [comment]
https://github.com/storybook-community/storybook-dark-mode should let you define two themes, one dark and one light. That should let you define two distinct brand image URLs.
